src/dfx-core/src/config/model/network_descriptor.rs

```diff
@@ -1,18 +1,31 @@
-use crate::config::model::dfinity::NetworkType;
-use crate::config::model::dfinity::{DEFAULT_IC_GATEWAY, DEFAULT_IC_GATEWAY_TRAILING_SLASH};
+use crate::config::model::dfinity::{
+    NetworkType, PlaygroundConfig, DEFAULT_IC_GATEWAY, DEFAULT_IC_GATEWAY_TRAILING_SLASH,
+};
 use crate::config::model::local_server_descriptor::LocalServerDescriptor;
 use crate::error::network_config::NetworkConfigError;
 use crate::error::network_config::NetworkConfigError::{NetworkHasNoProviders, NetworkMustBeLocal};
 use crate::error::uri::UriError;
 
+use candid::Principal;
 use slog::Logger;
 use std::path::{Path, PathBuf};
 use url::Url;
 
+//"mwrha-maaaa-aaaab-qabqq-cai"
```

Comment:
See https://github.com/dfinity/sdk/blob/master/src/dfx/src/lib/integrations/bitcoin.rs#L38 for an example of verifying this at test time rather than having a comment